apps/synapse-portal-framework/src/style/App.scss

```diff
@@ -0,0 +1,20 @@
+@use './variables' as Portal;
```

Comment:
Shared scss was slightly changed. Before, this file started with
```scss
// Load variable_overrides first, so correct values for variables will be applied to the rest of the styles
@use './config/style/variable_overrides';
```

and ended with 

```scss
// style_overrides should be last to ensure its styles have highest priority
@use './config/style/style_overrides';
```

This code was removed from the shared scss and moved to each portal's own scss.